pyhive/hive.py

```diff
@@ -252,6 +258,41 @@ def _fetch_more(self):
         for row in response.results.rows:
             self._data.append([_unwrap_col_val(val) for val in row.colVals])
 
+    def poll(self, fetch_logs=False):
```

Comment:
Could you document the return type?

(presto version: https://github.com/dropbox/PyHive/blob/master/pyhive/presto.py#L175)